src/WebJobs.Extensions.Http/Routing/WebJobsRouter.cs

```diff
@@ -51,12 +51,6 @@ private void InitializeRouteCollections()
 
         public Task RouteAsync(RouteContext context)
         {
-            // /admin/* routes should not be allowed to be overriden by proxies.
```

Comment:
Yes it should not get hit. If we left it here then I needed to add more logic for admin/warmup here because that's an exception. we can still add this logic in host repo under InvokeAsync in ScriptRouteHandler